### PR TITLE
Fix OrePAN2 tests on perl 5.10.0

### PR DESCRIPTION
**Pull request by davel**
Hi,

I have added a missing `use` and a version constraint which make OrePAN2 work on perl 5.10.0. I've also tweaked .travis.yml to work for me and also added a couple of dependencies which Kwalitee was complaining about.

All the best,
Dave

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,15 +5,7 @@ addons:
       - aspell
       - aspell-en
 before_install:
-  - 'cpanm --notest App::cpm'
```
Comment (oalders) [inline .travis.yml]: Why revert this change?  `cpm` installs in parallel.  This will make the Travis builds slower.

File: .travis.yml [hunk 2]
```diff
@@ -5,15 +5,7 @@ addons:
...
   - 'eval $(curl https://travis-perl.github.io/init) --auto'
-  - 'AUTHOR_TESTING=0 cpm install --workers $(test-jobs) --global Code::TidyAll::Plugin::SortLines::Naturally Code::TidyAll::Plugin::UniqueLines Perl::Tidy'
-cache:
-  directories:
-    - '~/perl5'
```
Comment (oalders) [inline .travis.yml]: Why remove the cache?  This will also slow down future builds.

File: .travis.yml [hunk 3]
```diff
@@ -26,11 +18,13 @@ matrix:
       perl: '5.28'
 notifications:
   email:
+    if: repo = "tokuhirom/OrePAN2"
     on_failure: always
     on_success: always
     recipients:
       - olaf@wundersolutions.com
 perl:
+  - '5.10.0'
```
Comment (oalders) [inline .travis.yml]: I don't think this is doing what was intended: https://travis-ci.org/tokuhirom/OrePAN2/jobs/622959535#L145-L239

Maybe we need to switch to `trusty`.  Some of the other Perls are also reverting to 5.22.1

File: cpanfile [hunk 4]
```diff
@@ -31,13 +31,15 @@ requires 'Ref::Util';
 requires 'Try::Tiny';
 requires 'Type::Params';
 requires 'Types::URI';
+requires 'feature';
```
Comment (oalders) [inline cpanfile]: I think rather than adding `feature` here, we can actually just remove the use of `feature` from the dist.  It's not bringing much to the table.

**Comment by oalders** [comment]
BTW, thanks for your work on this @davel.  We should probably do some follow-on work with the Travis config.  This has also helped expose a few more things that can be improved.

File: .travis.yml (hunk 1 above)
Comment (davel) [reply .travis.yml]: Hi @oalders, thank you for merging the change. I removed this line (and lines 10-13) to revert back to the travis-perl defaults. Travis was failing to build on all versions. See https://travis-ci.org/davel/OrePAN2/jobs/620791401 which attempts to build f32a01504832ee72c8077efd6b294dd67830ddaf, which was your most recent revision.

It used to work, https://travis-ci.org/tokuhirom/OrePAN2/builds/450006637 builds the exact same revision. I suspect something external.

File: .travis.yml (hunk 3 above)
Comment (davel) [reply .travis.yml]: Yes, I see what you mean.

However, this build, https://travis-ci.org/davel/OrePAN2/jobs/620867872 shows a failure exactly like I would expect for 5.10.0, yet 5.22.1 is reported. Compare with any of http://matrix.cpantesters.org/?dist=OrePAN2%200.47;perl=5.10.0;reports=1

Could the helper scripts somehow be reporting the wrong Perl yet running the correct one?

File: .travis.yml (hunk 1 above)
Comment (oalders) [reply .travis.yml]: Hi @davel.  Thanks for noticing that.  See https://github.com/travis-perl/helpers/issues/71  This needs to be fixed in the helpers or I think just using `dist: trusty` should get the older Perls working again.

File: .travis.yml (hunk 3 above)
Comment (oalders) [reply .travis.yml]: That's interesting.  I don't have a good answer for that, but I don't see anything in the log file that suggests it's 5.10 that is running the code.

Comment (davel) [reply .travis.yml]: Yes, it looks like Travis is asking the wrong Perl to report it's version. In a branch I added perl 5.8 to the list, and it complains about `use feature`, as I'd expect for 5.8. It does however report 5.22.1 in the prologue. See https://travis-ci.org/davel/OrePAN2/jobs/623314181

Comment (oalders) [reply .travis.yml]: Ah, I was wrong!  https://travis-ci.org/davel/OrePAN2/jobs/623314181#L283-L298  https://github.com/travis-perl/helpers#notes  I've been burned by this before.  :(  Thanks for pointing that out.  It's really confusing.
